[2023-08-22T08:58:51.177+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: csv_dag.counting_rows scheduled__2023-08-19T00:00:00+00:00 [queued]>
[2023-08-22T08:58:51.195+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: csv_dag.counting_rows scheduled__2023-08-19T00:00:00+00:00 [queued]>
[2023-08-22T08:58:51.196+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-08-22T08:58:51.219+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): counting_rows> on 2023-08-19 00:00:00+00:00
[2023-08-22T08:58:51.226+0000] {standard_task_runner.py:57} INFO - Started process 86030 to run task
[2023-08-22T08:58:51.232+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'counting_rows', 'scheduled__2023-08-19T00:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/csv_dag.py', '--cfg-path', '/tmp/tmpw59unqq7']
[2023-08-22T08:58:51.239+0000] {standard_task_runner.py:85} INFO - Job 55: Subtask counting_rows
[2023-08-22T08:58:51.330+0000] {task_command.py:410} INFO - Running <TaskInstance: csv_dag.counting_rows scheduled__2023-08-19T00:00:00+00:00 [running]> on host 908f67153615
[2023-08-22T08:58:51.469+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@my_first_dag.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='csv_dag' AIRFLOW_CTX_TASK_ID='counting_rows' AIRFLOW_CTX_EXECUTION_DATE='2023-08-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-19T00:00:00+00:00'
[2023-08-22T08:58:51.471+0000] {logging_mixin.py:150} INFO - An error occurred: [Errno 2] No such file or directory: 'downloaded.csv'
[2023-08-22T08:58:51.471+0000] {python.py:183} INFO - Done. Returned value was: -1
[2023-08-22T08:58:51.507+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=counting_rows, execution_date=20230819T000000, start_date=20230822T085851, end_date=20230822T085851
[2023-08-22T08:58:51.566+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-22T08:58:51.588+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
